chia/consensus/block_body_validation.py

```diff
@@ -407,16 +403,10 @@ async def validate_block_body(
 
     fees = removed - added
     assert fees >= 0
-    assert_fee_sum: uint128 = uint128(0)
-
-    for npc in npc_list:
-        if ConditionOpcode.RESERVE_FEE in npc.condition_dict:
-            fee_list: List[ConditionWithArgs] = npc.condition_dict[ConditionOpcode.RESERVE_FEE]
-            for cvp in fee_list:
-                fee = int_from_bytes(cvp.vars[0])
-                if fee < 0:
-                    return Err.RESERVE_FEE_CONDITION_FAILED, None
-                assert_fee_sum = uint128(assert_fee_sum + fee)
+    assert_fee_sum: uint64 = uint64(0)
```

Comment:
I think this needs to be uint128 since the max coin amount is uint64 but we might have multiple coins. not that anyone will pay a fee of < 18 million XCH though :P

Reply:
There's a consensus rule saying the sum of all `RESERVE_FEE` conditions may not exceed `uint64`. (technically, it says the sum of all fees aren't allowed to exceed uint64, but that implies that the reserve_fee also cannot exceed).

The current implementation enforcing this is here: https://github.com/Chia-Network/clvm_rs/blob/0.1.20/src/gen/conditions.rs#L435
The original implementation of this rule is here: https://github.com/Chia-Network/chia-blockchain/blob/1.0.0/src/consensus/block_body_validation.py#L355